SeleniumTask/src/test/java/LogPage.java

```diff
@@ -0,0 +1,20 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static constants.Constants.*;
+
+class LogPage {
+    private WebDriver driver;
+
+    LogPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    MainPage enterLoginAndPasswordAndSubmit() {
+        driver.findElement(By.id(USER_ID)).sendKeys(LOGIN);
```

Comment:
also you dont have a business object.
In page objects you only store locators and business objects use this locators to click(), sendKeys() etc..